pom.xml

```diff
@@ -190,6 +186,19 @@
 				<version>${jodatime}</version>
 				<scope>test</scope>
 			</dependency>
+
+			<dependency>
+				<groupId>org.apache.commons</groupId>
```

Comment:
Good catch.  I added the test scope to the dependencies that I had to add to get the tests working as a result of changes in the upstream dependencies.  Does that suffice?  The deps are 100% required for the tests to compile and run.